packages/dapp/src/components/Widgets/Widgets.tsx

```diff
@@ -56,37 +63,70 @@ export function DualWidget() {
     }
   }, [activeTab, onChangeTab, starterVariant, starterVariantUsed]);
 
-  useEffect(() => {
+  const handleGetStarted = async (event) => {
+    const classList = event?.target?.classList;
+    if (
+      classList?.contains?.('stats-card') ||
+      classList?.contains?.('link-lifi')
+    ) {
+      return 0;
+    } else {
+      event.stopPropagation();
+      setShowWelcome(false);
+      await setTimeout(() => {}, 600);
```

Comment:
`await` has no effect here and I'm curious to know why do we even need this 😄

Reply:
Well, thanks! A little memorial of those times when I had an extra "showFadeOut" - state. I am ready to forget about it and clean it up.